Update bigquery-setup.md (#3980)
fishtown-analytics/docs.getdbt.com: website/docs/docs/core/connect-data-platform/bigquery-setup.md

:::
  

> No timeout is set by default. (For historical reasons, some query types use a default of 300 seconds when the `job_execution_timeout_seconds` configuration is not set.) When `job_execution_timeout_seconds` is set, if any dbt query, takes longer than 300 seconds to return a result, the dbt-bigquery adapter will run into an exception:

Reviewer: ```suggestion
No timeout is set by default. (For historical reasons, some query types use a default of 300 seconds when the `job_execution_timeout_seconds` configuration is not set). When you do set the `job_execution_timeout_seconds`, if any dbt query takes more than 300 seconds to finish, the dbt-bigquery adapter will run into an exception:
```

```
 Operation did not complete within the designated timeout.
```

:::caution Note

> The `job_execution_timeout_seconds` represents the number of seconds to wait for the [underlying HTTP transport](https://cloud.google.com/python/docs/reference/bigquery/latest/google.cloud.bigquery.job.QueryJob#google_cloud_bigquery_job_QueryJob_result). It DOES NOT represent the maximum allowable time that a BigQuery job can run for within BigQuery itself - this means that even in the scenario above, where dbt-bigquery ran into an exception at 300 seconds, the BigQuery job or query could still be running in BigQuery up to the BigQuery default query job timeout configuration.

Reviewer: trying to make this more concise and remove the multiple uses of bigquery. does this sound right @jeremyyeo ?
```suggestion
The `job_execution_timeout_seconds` represents the number of seconds to wait for the [underlying HTTP transport](https://cloud.google.com/python/docs/reference/bigquery/latest/google.cloud.bigquery.job.QueryJob#google_cloud_bigquery_job_QueryJob_result). It _doesn't_ represent the maximum allowable time for a BigQuery job itself. So, if dbt-bigquery ran into an exception at 300 seconds, the actual BigQuery job could still be running for the time set in BigQuery's own timeout settings.
```
